### PR TITLE
Add EntityHider (for Practice and MiniGames servers)

### PR DESCRIPTION
**Pull request by SaithTime**
Added an entity hider system if the shooter is not visible by other.

Method based on a private patched version of SpigotX

### CODE REVIEW

**Review by Elierrr — CHANGES_REQUESTED** [review]
Add a configuration option.

File: pom.xml [hunk 1]
```diff
@@ -22,6 +22,10 @@
         <module>NachoSpigot-API</module>
     </modules>
 
+    <build>
```
Comment (CyberFlameGO) [inline pom.xml:25]: Not everyone needs the API built, so in my opinion this is pretty pointless

**Review by CyberFlameGO — CHANGES_REQUESTED** [review]
I agree with Elier, and it's easy to make this a configuration option

File: pom.xml (hunk 1 above)
Comment (CyberFlameGO) [reply pom.xml:25]: Nevermind iirc the server needs the api to build upon

**Comment by CyberFlameGO** [comment]
You could also add `clean` (**NOT REPLACE**, so just make it `clean install` instead of just `install`) to the [server module](https://github.com/CobbleSword/NachoSpigot/blob/master/NachoSpigot-Server/pom.xml) default goals, and fix the [README.md](https://github.com/CobbleSword/NachoSpigot/blob/master/README.md) instructions. You may even be able to just add the default goals to the [parent POM](https://github.com/CobbleSword/NachoSpigot/blob/master/README.md) instead of messing around with each module

**Comment by Lucaskyy** [comment]
Superseded by #213

**Pull request closed without merging**